packages/graBing/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname="graBing"
+pkgver=9.bb8b08b
+pkgrel=1
+pkgdesc="Counts all the hostnames for an IP adress"
+arch=('x86_64' 'i686')
+url="https://github.com/black-brain/graBing"
+license=('GPL3')
+groups=('blackarch-fingerprint' 'blackarch-scanner')
+source=('git+https://github.com/black-brain/graBing')
+depends=('python2')
+makedepends=('git')
+provides=("graBing")
```

Comment:
Not sure if needed, can you elaborate more?
(the same goes for `conflicts` and `replaces`).

Reply:
My bad, I will fix it.